pkg/testers/ginkgo/ginkgo.go

```diff
@@ -94,9 +101,109 @@ func (t *Tester) pretestSetup() error {
 	}
 	log.Printf("Using kubeconfig at %s", t.kubeconfigPath)
 
+	if err := t.AcquireTestPackage(); err != nil {
+		return fmt.Errorf("failed to get ginkgo test package from published releases: %s", err)
+	}
+
 	return nil
 }
 
+// TODO(michaelmdresser): change behavior if a local built e2e.test package
+// is available
```

Comment:
nit: also add functionality download a specific version instead of just latest.

Reply:
Done.